sapi/cli/php_cli_server.c

```diff
@@ -2310,7 +2312,13 @@ static int php_cli_server_recv_event_read_request(php_cli_server *server, php_cl
 	char *errstr = NULL;
 	int status = php_cli_server_client_read_request(client, &errstr);
 	if (status < 0) {
-		php_cli_server_logf("%s Invalid request (%s)", client->addr_str, errstr);
+		if (strcmp(errstr, php_cli_server_request_error_unexpected_eof) == 0 && client->parser.state == s_start_req) {
+#if PHP_DEBUG
```

Comment:
This part would mean - if debug is not enabled and condition evaluates to false, no error message is shown. Is that intentional?

Reply:
Yes, that is the intended behavior. :)